### DISCUSSION TITLE
Create some criteria for adding new distros to quickget

### DISCUSSION THREAD

**Discussion by flexiondotorg** [quickemu-project/quickemu#811] [Ideas]
Adding every Linux distro to `quickget` doesn't add value but does introduce a maintenance burden for the project.

Therefore, let's create some criteria for Linux distros to meet to be added. Once we agree on these criteria, they should also be retrospectively applied to remove distros from `quickget` that do not meet them.

This only applies to Linux distros because

- There are fewer BSDs, and they are distinctive
- macOS and Windows are just two OSs and widely used
- non-Linux operating systems are attractive for that reason alone and warmly welcomed

Here are my opening suggestions:

- A proven track record of regular releases
- A means to verify the download integrity via a hash or signature
- Respins of existing distributions must be official projects of the parent distro, be a top-tier distro in their own right or have a demonstrable and distinctive capability

Suggestions for better criteria is welcome :bow:

**Comment by zen0bit** [comment]
Operating systems not intended for merging in quickemu: (comments by @flexiondotorg )
1. AgarimOS #753 - I am going to politely decline this one. Another of the endless Void respins. It changes often and will be a maintenance burden.
2. Cereus linux #659 - I am not a fan of adding endless forks of Void Linux.
3. CROWZ #796 - Politely decline this one. First release of a Debian/Devuan respin.
4. DietPi #660 - This is primarily focused on SBCs. Add to quickemu for use in a VM doesn't add much value IMO.
5. F-Void linux #661 - I am not a fan of adding endless forks of Void Linux.
6. gabeeOS #671 - I am not a fan of adding endless forks of Void Linux.
7. PureOS #786 - I am not going to accept distros that are primarily designed for/by OEMs as they will often carry hardware-specific enablement that is unnecessary in VMs.
8. TUXEDO OS #749 - I am not going to accept distros that are primarily designed for/by OEMs as they will often carry hardware-specific enablement that is unnecessary in VMs.
9. VoidPup #663 - I am not a fan of adding endless forks of Void Linux.
10. Zephix #795 - A live-only distribution. I'm not compelled to add this to quickemu.

Is deprecated now:
1. Miyo Linux #752 - I am going to politely decline this one. It is a fringe Devuan respin. I think we need to create some guidelines for what distros will be added, because quickget becoming "install distrowatch" is not going to be sustainable.

PR to be merged:
1. #!++ #881
3. Artix Linux #758
4. Athena OS #750
5. bazzite #797
13. biglinux #748
14. Chimera Linux #781
15. GNOME OS #913
16. Guix #768
17. Nitrux #873
18. Parrot Security #893
19. Primtux #887
20. Slax #761
21. Slint #889
22. SliTaz GNU/Linux #664
23. SparkyLinux #799

In PR but not yet finished:
1. AROS One #712
2. icaros desktop #713
3. whonix #756
